tests/v1/kv_connector/toy_proxy_server.py

```diff
@@ -0,0 +1,145 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
note: this is needed so we can run in the CI (since the other one is in examples)